files/internals/internals.conf

```diff
@@ -27,6 +27,7 @@ wget_retries="3"
 wget_proxy=""
 
 wget=`which wget 2> /dev/null`
+wgetopt=`-e http_proxy="$wget_proxy" -e https_proxy="$wget_proxy" -T$wget_timeout -t$wget_retries -q`
```

Comment:
wgetopt= is defined with backticks = will try run as a command instead of defining as a variable